lib/utils/KalturaUtils.js

```diff
@@ -466,6 +466,19 @@ KalturaUtils = {
 		});
 
 		request.end();
+	},
+
```

Comment:
Then this will be replaced by a NULL pointer exception since several instances that call the code do not handle NULL - this is why I prefer to throw the exception if requested by the caller

From: yuly-roberman [mailto:notifications@github.com]
Sent: Monday, March 7, 2016 12:02 PM
To: kaltura/play-server <play-server@noreply.github.com>
Cc: Elad Cohen <elad.cohen@kaltura.com>
Subject: Re: [play-server] Fixed calls to JSON parse where input invalid (#238)


In lib/utils/KalturaUtils.js<https://github.com/kaltura/play-server/pull/238#discussion_r55182223>:

> @@ -466,6 +466,19 @@ KalturaUtils = {

>              });

>

>              request.end();

> +     },

> +

no need to throw an exception, just log the relevant error to error log

—
Reply to this email directly or view it on GitHub<https://github.com/kaltura/play-server/pull/238/files#r55182223>.